Framework/Adxstudio.Xrm/Activity/ActivityDataAdapter.cs

```diff
@@ -13,7 +13,7 @@ namespace Adxstudio.Xrm.Activity
 	using Adxstudio.Xrm.Security;
 	using Adxstudio.Xrm.Services.Query;
 	using Adxstudio.Xrm.Notes;
-using Adxstudio.Xrm.Services;
+    using Adxstudio.Xrm.Services;
```

Comment:
Sorry for being nit picky, but can you please change the indentation to a tab to be consistent with the other code?